check-your-phone.md

```diff
@@ -0,0 +1,2 @@
+You look up and suddenly, there is a woman in 
```

Comment:
@ChristineSchatz, if you wish your change to appear in the other file instead, you should also remove this file from the pull request, since otherwise both files would be added to the story, which would clutter up the repository.